styles/styles.css

```diff
@@ -318,3 +318,100 @@ main .section {
 main .section.highlight {
   background-color: var(--highlight-background-color);
 }
+
+.section.secondary-banner {
+  display: flex;
+  min-width: 0;
+  max-width: 100%;
+  flex-flow: row nowrap;
+  justify-content: center;
+  padding-top: 66px;
+  padding-bottom: 72px;
+  background-color: #00ada2;
+}
+
+.section.secondary-banner .default-content-wrapper {
+  display: flex;
+  flex-direction: column;
+  min-width: 0;
+  max-width: 100%;
+  max-width: 1280px;
+  margin-right: auto;
+  margin-left: auto;
+  text-align: center;
+}
+
+.section.secondary-banner .default-content-wrapper > p:first-child {
+  font-family: var(--body-font-family);
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 1.25;
+  color: #fff;
+  letter-spacing: 3px;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.section.secondary-banner .default-content-wrapper > p:nth-child(3) {
```

Comment:
rather than adjusting p:nth-child, would it make sense to adjust the source content for authors such that instead of '**customer story**' as a <p> content, we convert the source content and make it  general that any h3, h4, h5 of the secondary banner would be this specific style

Reply:
This is done!